src/action.ts

```diff
@@ -71,9 +71,9 @@ async function fetchOrLocateContrastResultsFile() {
   throw new Error("Contrast requires a file to be provided");
 }
 
-async function fetchOrLocateSonarResultsFile(resultType : SONAR_RESULT) {
+async function fetchOrLocateSonarResultsFile(resultType : SONAR_RESULT, index ?: number) {
```

Comment:
Is this method actually called with this `index` parameter anywhere? I'm not sure I understand the reason for adding it.

Reply:
no, it was for testing purposes when I was uploading multiple files